.travis.yml

```diff
@@ -237,10 +245,20 @@ matrix:
       env: SWIGLANG=tcl
       sudo: required
       dist: trusty
+    - compiler: gcc
+      env: SWIGLANG=fortran SWIG_FC=gfortran-5 # Note: C++98
+      sudo: required
+      dist: trusty
+      addons: *gfortran5
```

Comment:
Could you remove the addons in this file and make equivalent changes in Tools/travis-linux-install.sh. This keeps consistency with other languages. The main reason it is done like this is to make it clear what commands need to be run in a non-Travis environment to replicate what needs to be installed on an Ubuntu box to replicate the Travis environment. What gets installed should be driven by SWIGLANG and VER. I'm not 100% sure, if you can  get rid of the Fortran specific name SWIG_FC though.

Reply:
Aha, that makes a lot more sense -- I hadn't looked at that file very carefully. I've gotten rid of `SWIG_FC` but it is still necessary to export `$FCFLAGS` to the build: is the way I'm doing that acceptable?